### DISCUSSION TITLE
Add custom variable to default_service_columns and change name

### DISCUSSION THREAD

**Discussion by Krounth** [sni/thruk#1233] [Q&A] [answered]
Hi,
Im triying add my custom var to default_service_columns to display it to all users and change the name to delete the _ showed in the table.
I try to add it but not are working.
`show_custom_vars=_CRITICIDAD`
`default_service_columns = host_name,description,state,last_check,duration,current_attempt,_CRITICIDAD:Criticidad,plugin_output,site`

What am i doing wrong?

**Comment by sni** [comment, marked as answer]
this has been reworked lately and should work better with next release. If you like, you could already try the current daily builds.
